app/controllers/application_controller.rb

```diff
@@ -35,6 +35,17 @@ def require_rvt_leader
     unauthorized_page unless current_user.roles.rvt_leader?
   end
 
+  def require_resort_leader
+    @own_membership = current_user.roles.resort_leader?(current_group)
```

Comment:
I am not sure, it is a good workaround to create a `@own_membership`, because the function suggests a role check, not something like `current_membership`.

This could be solved easily by something like `unauthorized_page unless current_user.roles.resort_leader?(current_group)`